features/language-core.feature

```diff
@@ -214,7 +214,7 @@ Feature: Manage translation files for a WordPress install
       """
     And the return code should be 0
 
-  @require-php-5.6 @less-than-php-7.0
+  @require-wp-4.0 @less-than-php-7.0
```

Comment:
I removed it `@less-than-php-7.0` and lots of warnings have appeared in the tests.

Reply:
Ah, I see, it's because the test is downgrading to WP 4.8 and  4.0.1 respectively. Makes sense that these potentially throw warnings on newer PHP versions.

Reply:
Let's add a comment to clarify this here for future readers